### PR TITLE
fix(monitoring): make VPCSC env comparison case-insensitive

### PR DESCRIPTION
**Pull request by steinwaywhw**
The actual environment variable value may be `True`, instead of `true`. So just make it case insensitive.

### CODE REVIEW

File: monitoring/noxfile.py [hunk 1]
```diff
@@ -122,7 +122,8 @@ def system(session):
     session.install("-e", ".")
 
     # Additional setup for VPCSC system tests
-    if os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC") != "true":
+    in_vpc = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC")
```
Comment (tseaver) [inline monitoring/noxfile.py]: We can avoid the special handling for `None` below by providing a default, e.g.:
```suggestion
    in_vpc = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "false")
```

File: monitoring/noxfile.py [hunk 2]
```diff
@@ -122,7 +122,8 @@ def system(session):
...
+    if not in_vpc or in_vpc.lower() != "true":
```
Comment (tseaver) [inline monitoring/noxfile.py]:
```suggestion
    if in_vpc.lower() != "true":
```

**Comment by steinwaywhw** [comment]
@googlebot I fixed it.

**Comment by steinwaywhw** [comment]
Hi tseaver@, please sign CLA using your personal email. See link above, or equivalently [this one](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleapis%2Fgoogle-cloud-python%2Fpull%2F9564)

**Comment by tseaver** [comment]
@googlebot I consent.

**Comment by tseaver** [comment]
@steinwaywhw I made those edits through the web:  the e-mail associated with them (`tseaver@palladion.com`) *is* my normal one.

**Pull request merged**